ss58-registry-derive/Cargo.toml

```diff
@@ -10,10 +10,14 @@ description = "Registry of wellknown SS58 id types"
 proc-macro = true
 
 [dependencies]
```

Comment:
https://github.com/paritytech/ss58-registry/pull/6#discussion_r707568038

Reply:
I see what you mean. That works pretty well.